airbyte-webapp/src/components/connection/CatalogTree/next/NextTable/NextTable.module.scss

```diff
@@ -0,0 +1,73 @@
+@use "scss/colors";
+
+// ------- <table/> --------
+.table {
+  border-spacing: 0;
+  width: 100%;
+  max-width: 100%;
+  border-radius: 10px;
```

Comment:
Could you use variable **$border-radius-md: 10px;** from _variables.scss?

Reply:
oh, yes, missed that. Thanks!